cwf/gateway/integ_tests/pipelined.yml

```diff
@@ -37,7 +37,7 @@ static_services: [
   'access_control',
   'tunnel_learn',
   'vlan_learn',
-  'li_mirror',
+  #'li_mirror',
```

Comment:
Fmu what is/was li_mirror?

Reply:
Its essentially the traffic mirroring controller but I'm disabling it for now as It needs some modifications